docs/acp/user/intro-to-acp/developing-an-app.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Developing an app
+Descriptions: Learn how to develop an app
+template: howto-guide-template
+---
+
+To develop an app, follow the instructions in this document.
+
+## Prerequisites
+
+- You have completed the [thought process](#thought-process) for your app.
+- You have [installed the Spryker SDK](https://docs.spryker.com/docs/sdk/dev/spryker-sdk.html#installation).
+- You have an empty GitHub repository.
+- You have a local project directory where you want to work, for example, `/www/my-app`.
+ 
+## Thought process
+First, think about what your app should be capable of: what features it will bring, and what data will be exchanged, not only to you but also to those interested in your app functionality. For example, what messages could be of interest to others, and what API endpoints you should provide. 
+
+### API-first
+It's strongly recommended that apps follow the API-first approach. 
+API-first means that your app is centered on the API. It should be possible to perform every action via the scripting language, and every piece of functionality should be available for other systems to leverage. For more information on the API-first approach, see [this blog post](https://www.algolia.com/blog/product/the-5-principles-of-api-first-development-and-what-does-api-first-even-mean/).
+You need to have a clear understanding of what your app API will provide to others, and always keep that in mind when you design your app.
+
+### Schema-first
+
+Before you start with the development, you should design your API schema files first. Depending on your requirements, you can have an OpenAPI or an AsyncAPI schema file. In this step, you define the SyncAPI endpoints your app will provide to others, the messages you will emit or consume, and the data you expect to work with.
+
+For more information about AsyncAPI schema design, see [Designing your APIs with AsyncAPI](https://www.asyncapi.com/blog/designing_your_apis_with_asyncapi_part_1). 
+
+For more information about OpenAPI schema design, see [Best practices in API design](https://swagger.io/resources/articles/best-practices-in-api-design/).
+
+You can use the following tools to design your APIs:
+- [AsyncAPI Studio](https://studio.asyncapi.com/)
+- [Swagger Editor](https://editor.swagger.io/)
+
+You can also use the wizards provided by Spryker, which will be used by the [SprykerSDK workflow](LINK).
+
+## Build an app
+
+You can build a new app with the help of the SprykerSDK. After you have installed the Spryker SDK, you can run the following commands to start building the app:
+
+{% info_block infoBox "Info" %}
+
+You can skip through the prompts by hitting Enter.
+
+{% endinfo_block %}
+
+```bash
+cd /www/my-app (The local project directory you created before)
+spryker-sdk sdk:init:project --workflow=pbc
+spryker-sdk sdk:workflow:run
+```
+The first line ensures that you are in the correct working directory. The second line initializes the project, and the third line runs the workflow.
```

Comment:
makes sense to split code blocks into two and comment them separately, instead of using first line, second line